apps/daimo-mobile/src/view/screen/HistoryOpScreen.tsx

```diff
@@ -137,123 +147,109 @@ function TransferBody({
   account: Account;
   op: TransferOpEvent;
 }) {
-  const opRequestId = op.nonceMetadata
-    ? DaimoNonceMetadata.fromHex(op.nonceMetadata)?.identifier.toString()
-    : undefined;
-  const matchingTrackedRequest = account.trackedRequests.find(
-    (req) =>
-      req.requestId === opRequestId &&
-      req.amount === `${op.amount}` &&
-      op.to === account.address
-  );
+  // TODO: show if this transfer filled our request.
```

Comment:
this is not currently in the designs

leaving the logic here for if we re-add it.

Reply:
doesn't need to be in this pr but imo we can just drop all the current request logic and redo it later when we make requests compatible with non-daimo wallets for the sender.